### ISSUE TITLE
Unable to understand 1e9 in the validation loss

### ISSUE THREAD

**Issue by thak123** [francoisgermain/SpeechDenoisingWithDeepFeatureLosses#5]
why there is a 1e9 multiplied in the loss format string and it is absent for the train loss
    
`  
  ### PRINT EPOCH TRAINING LOSS AVERAGE
    str = "T: %d\t " % (epoch)
    if SE_LOSS_TYPE == "FL":
        for j in range(SE_LOSS_LAYERS + 1):
            str += ", %10.6e" % (np.mean(loss_train, axis=0)[j])
    else:
        str += ", %10.6e" % (np.mean(loss_train, axis=0)[0])
`

`
    ### PRINT VALIDATION EPOCH LOSS AVERAGE

    str = "V: %d " % (epoch)
    if SE_LOSS_TYPE == "FL":
        for j in range(SE_LOSS_LAYERS + 1):
             str += ", %10.6e" % (np.mean(loss_val, axis=0)[j] * 1e9) 
    else:
        str += ", %10.6e" % (np.mean(loss_val, axis=0)[0] * 1e9)`
